bentoml/_internal/frameworks/FRAMEWORK_TEMPLATE_PY

```diff
@@ -23,10 +23,9 @@ try:
     import FRAMEWORK_PY
 except ImportError:  # pragma: no cover (trivial error checking)
     raise MissingDependencyException(
-        """FRAMEWORK_PY is required in order to use module `bentoml.lightgbm`, install
-        FRAMEWORK_PY with `pip install FRAMEWORK_PY`. For more information, refer to
-        <FRAMEWORK INSTALL URL>
-        """
+        "`FRAMEWORK_PY` is required in order to use module `bentoml.lightgbm`, install "
```

Comment:
Current format:

<img width="853" alt="Screen Shot 2022-07-03 at 3 09 48 AM" src="https://user-images.githubusercontent.com/489344/177035031-56b4fe47-c6cf-4523-acbe-e55117cfe572.png">

After this change:
<img width="1384" alt="Screen Shot 2022-07-03 at 3 10 18 AM" src="https://user-images.githubusercontent.com/489344/177035044-2a6e38aa-173a-4679-8e6d-0dff1ead0349.png">